boefjes/boefjes/plugins/pdio_subfinder/main.py

```diff
@@ -0,0 +1,16 @@
+from typing import Tuple, Union, List
+
+import docker
+
+from boefjes.job_models import BoefjeMeta
+
+SSLSCAN_IMAGE = "projectdiscovery/subfinder:latest"
```

Comment:
This was probably copy-pasted from another boefje? `SSLSCAN_IMAGE` => `SUBFINDER_IMAGE`?